x-pack/plugin/deprecation/src/test/java/org/elasticsearch/xpack/deprecation/IndexDeprecationChecksTests.java

```diff
@@ -7,248 +7,31 @@
-        assertEquals(singletonList(expected), issues);
-    }
-
-    public void testAllMetaFieldIsDisabledByDefaultCheck() throws IOException {
-        XContentBuilder mapping = XContentFactory.jsonBuilder();
-        mapping.startObject(); {
-            mapping.startObject("_all"); {
-                mapping.field("enabled", randomFrom("1", 1, "true", true));
-            }
-            mapping.endObject();
-        }
-        mapping.endObject();
-
-        IndexMetaData indexMetaData = IndexMetaData.builder("test")
-            .putMapping("testAllEnabled", Strings.toString(mapping))
-            .settings(settings(Version.V_5_6_0))
-            .numberOfShards(1)
-            .numberOfReplicas(0)
-            .build();
-
-        DeprecationIssue expected = new DeprecationIssue(DeprecationIssue.Level.INFO,
-            "The _all meta field is disabled by default on indices created in 6.0",
-            "https://www.elastic.co/guide/en/elasticsearch/reference/6.0/" +
-                "breaking_60_mappings_changes.html#_the_literal__all_literal_meta_field_is_now_disabled_by_default",
-            "types: [testAllEnabled]");
-        List<DeprecationIssue> issues = DeprecationChecks.filterChecks(INDEX_SETTINGS_CHECKS, c -> c.apply(indexMetaData));
-        assertEquals(singletonList(expected), issues);
-    }
-
-    public void testIncludeInAllCheck() throws IOException {
-        XContentBuilder mapping = XContentFactory.jsonBuilder();
-        mapping.startObject(); {
-            mapping.startObject("_all"); {
-                mapping.field("enabled", false);
-            }
-            mapping.endObject();
-            mapping.startObject("properties"); {
-                mapping.startObject("my_field"); {
-                    mapping.field("type", "text");
-                    mapping.field("include_in_all", false);
-                }
-                mapping.endObject();
-            }
-            mapping.endObject();
-        }
-        mapping.endObject();
-
-        IndexMetaData indexMetaData = IndexMetaData.builder("test")
-            .putMapping("testIncludeInAll", Strings.toString(mapping))
-            .settings(settings(Version.V_5_6_0))
-            .numberOfShards(1)
-            .numberOfReplicas(0)
-            .build();
-
         DeprecationIssue expected = new DeprecationIssue(DeprecationIssue.Level.CRITICAL,
-            "The [include_in_all] mapping parameter is now disallowed",
-            "https://www.elastic.co/guide/en/elasticsearch/reference/6.0/" +
-                "breaking_60_mappings_changes.html#_the_literal_include_in_all_literal_mapping_parameter_is_now_disallowed",
-            "[[type: testIncludeInAll, field: my_field]]");
+            "Index created before 6.0",
+            "https://www.elastic.co/guide/en/elasticsearch/reference/master/" +
+                "breaking-changes-7.0.html#breaking_70_packaging_changes",
```

Comment:
```suggestion
                "breaking-changes-7.0.html",
```

Same reasoning as above.